pygrocy/grocy_api_client.py

```diff
@@ -288,8 +288,8 @@ class TransactionType(Enum):
 
 
 class GrocyApiClient(object):
-    def __init__(self, base_url, api_key, verify_ssl = True):
-        self._base_url = base_url
+    def __init__(self, base_url, api_key, port: int = 9192, verify_ssl = True):
```

Comment:
Could you store this in a constant so changing the default port is done in one place instead of having to touch multiple places.